io/plugins/eu.esdihumboldt.hale.io.topojson/src/eu/esdihumboldt/hale/io/topojson/TopoJsonInstanceWriter.java

```diff
@@ -260,33 +259,17 @@ else if (textValue.equals("null")) {
 								// Step 2: Convert java.util.Date to
 								// java.time.Instant
 								Instant instant = wrongDateFormat.toInstant();
-
-								// Step 3: Convert java.time.Instant to
-								// java.time.LocalDateTime
-								LocalDateTime localDateTime;
-								if (wrongDateFormat.toString().contains("CET")) {
-									// Convert the Date object to LocalDateTime
-									localDateTime = LocalDateTime
-											.ofInstant(wrongDateFormat.toInstant(), ZoneOffset.UTC);
-
-								}
-								else {
-									localDateTime = instant.atZone(ZoneId.systemDefault())
-											.toLocalDateTime();
-								}
+								// assume only date because of default behavior of Shapefile writer
+								LocalDate localDate = instant.atZone(ZoneId.systemDefault())
+										.toLocalDate();
 
 								// Get the year from LocalDateTime
-								int year = localDateTime.getYear();
-								if (year == 2) {
+								int year = localDate.getYear();
+								if (year == 2) { // XXX why this condition?
```

Comment:
I remember that by default, if the year was not set up, the `getYear()` would return 2.

Reply:
Thanks for the info, modified the comment.